redirects.yaml

```diff
@@ -1,39 +1,24 @@
-2009/02/?: "https://insights.ubuntu.com"
-2009/07/03/en-gran-canaria/?: "https://insights.ubuntu.com"
-2010/02/?: "https://insights.ubuntu.com"
-2010/10/?: "https://insights.ubuntu.com"
-2010/11/?: "https://insights.ubuntu.com"
-2011/09/juju-in-11-10-beta2-lxc-and-openstack-improvements/?: "https://insights.ubuntu.com"
-2012/02/juju-charm-school-webinar-march-8th/?: "https://insights.ubuntu.com"
-2012/04/announcing-the-uds-juju-charm-contest-for-uds-attendees/?: "https://insights.ubuntu.com"
-2012/11/dellworld/?: "https://insights.ubuntu.com/event/dellworld/"
-2012/11/hpdiscover/?: "https://insights.ubuntu.com/event/ubuntu-at-hp-discover-2012-2/"
-2012/12/?: "https://insights.ubuntu.com"
-2013/01/press-pack-ubuntu-phone/?: "https://insights.ubuntu.com/2013/06/27/ubuntu-phone-press-ki/"
-2013/02/dellfrs/?: "https://insights.ubuntu.com/event/dellfrs/"
-2013/02/press-pack-ubuntu-tablet/?: "https://insights.ubuntu.com/2013/02/01/ubuntu-tablet-press-pack/"
-2013/02/press-pack-ubuntu-tablet/faq/?: "https://insights.ubuntu.com/press-centre/"
-2013/04/17/filling-the-gaps/?: "https://insights.ubuntu.com"
-2013/05/08/people-behind-ubuntu-quality-howard-2/?: "https://insights.ubuntu.com"
-2013/07/edge-founders/?: "https://insights.ubuntu.com/2013/08/22/ubuntu-edge-thank-you/"
-2013/07/press-pack-ubuntu-edge/?: "https://insights.ubuntu.com/press-centre/"
-2014/02/press-pack-mwc-2014/?: "https://insights.ubuntu.com/2014/02/01/mwc-2014-online-press-pack/"
-2014/03/vmware-openstack-webinar/?: "https://insights.ubuntu.com/2014/03/18/customer-stories-from-openstackvmware-projects/"
+2009/.+/?: "https://insights.ubuntu.com"
+2010/.+/?: "https://insights.ubuntu.com"
+2011/.+/?: "https://insights.ubuntu.com"
+2012/.+/?: "https://insights.ubuntu.com"
+2013/.+/?: "https://insights.ubuntu.com"
```

Comment:
We need `2014` to catch: 
- 2014/02/press-pack-mwc-2014/
- 2014/03/vmware-openstack-webinar/

Reply:
that's my fault

the '?' makes it a non-greedy search... so remove the ? and it should work?

```
2009/.+/: "https://insights.ubuntu.com"
2010/.+/: "https://insights.ubuntu.com"
2011/.+/: "https://insights.ubuntu.com"
2012/.+/: "https://insights.ubuntu.com"
2013/.+/: "https://insights.ubuntu.com"
2014/.+/: "https://insights.ubuntu.com"
```

*Note: there might be more*

Reply:
Fixed